performance_test/src/experiment_execution/analyze_runner.cpp

```diff
@@ -54,6 +75,37 @@ AnalyzeRunner::AnalyzeRunner()
     m_sub_runners.push_back(DataRunnerFactory::get(m_ec.topic_name(), m_ec.com_mean(),
       RunType::SUBSCRIBER));
   }
```

Comment:
I guess the parenthesis shouldn't be closed here since we have other things to initialize in the constructor below this?

Reply:
The parenthesis you are pointing at is from for() loop, not from constructor, so we have all we need in the parentheses after #endif (line 56)

Reply:
Okay great. Thanks